packages/transactions/README.md

```diff
@@ -62,7 +62,8 @@ const transaction = await makeSTXTokenTransfer(txOptions);
 const serializedTx = transaction.serialize().toString('hex');
 
 // broadcasting transaction to the specified network
-broadcastTransaction(transaction, network);
+const broadcastResponse = await broadcastTransaction(transaction, network);
+const txId = broadcastResponse.txid;
```

Comment:
I think we should omit parsing out the `txId` in this and the following examples

```suggestion
```

Reply:
why? I think it adds valuable info (the return is a JSON object with `txid` as a property)